components/organisms/Layout.js

```diff
@@ -0,0 +1,117 @@
+import PropTypes from "prop-types";
+import Head from "next/head";
+import { ActionButton } from "../atoms/ActionButton";
+import { SearchBar } from "../atoms/SearchBar";
+import { Banner } from "../atoms/Banner";
+
+export const Layout = ({
+  bannerText,
+  bannerTitle,
+  children,
+  bannerStartTestingText,
+  bannerBecomeTesterText,
+}) => {
+  return (
+    <div>
+      <Head>
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+      </Head>
+      <div className="overflow-x-hidden">
+        <header>
+          <div className="lg:container xxs:px-6 xs:px-0 xs:mx-6 lg:mx-auto lg:px-6 xxl:mx-auto">
```

Comment:
Two points here.


1. At the "xs" breakpoint, we have "6" horizontal margin and "0" padding, and then once we get to "xxs", we add "6" horizontal padding and the margin disappears. It seems like we could delete `xxs:px-6` and then use `xs:px-0 xxs:mx-6`.
2. The `lg:container` class sets breakpoints but there are still 7 classes that we're adding to 4 different body elements. Can we roll them up into one class by combining them so we have less copy+paste?